Options.py

```diff
@@ -20,6 +21,15 @@
     import pathlib
 
 
+class Visibility(enum.IntFlag):
+    none = 0b0000
+    template = 0b0001
+    simple_ui = 0b0010  # show option in simple menus, such as player-options
+    complex_ui = 0b0100  # show option in complex menus, such as weighted-options
+    spoiler = 0b1000
+    all = 0b1111
```

Comment:
wouldn't be "not_simple_ui" a common value?

Reply:
Visibility.all - Visibility.spoiler may also be a common use case, for options that only have visual effects on the game and have no effect on generation. (Flashing reduction, sprite choice, etc.)

It would certainly be nice to clean up the spoiler log a little bit.

Reply:
SC2 player colors are visual-only too

Reply:
I'd say it's easy enough to get "all except X" with `Visibility.all ^ Visibility.spoiler`, to use spoiler for an example. I'd be fine to leave this as-is.